product_code/user_service/api/src/main/scala/de/upb/cs/uc4/user/api/UserService.scala

```diff
@@ -31,28 +31,31 @@ trait UserService extends UC4Service {
   def getUser(username: String): ServiceCall[NotUsed, User]
 
   /** Get all users from the database */
-  def getAllUsers(usernames: Option[String]): ServiceCall[NotUsed, GetAllUsersResponse]
+  def getAllUsers(usernames: Option[String], only_active: Option[Boolean]): ServiceCall[NotUsed, GetAllUsersResponse]
```

Comment:
```suggestion
  def getAllUsers(usernames: Option[String], onlyActive: Option[Boolean]): ServiceCall[NotUsed, GetAllUsersResponse]
```
We are using camelCase